core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -523,23 +523,23 @@ oppia.controller('LearnerDashboard', [
         $scope.subscriptionsList = (
           responseData.subscription_list
         );
-        $scope.numberDeletedIncompleteExplorations = (
-          responseData.number_of_deleted_activities.incomplete_explorations
+        $scope.numberNonexistentIncompleteExplorations = (
+          responseData.number_of_nonexistent_activities.incomplete_explorations
         );
-        $scope.numberDeletedIncompleteCollections = (
-          responseData.number_of_deleted_activities.incomplete_collections
+        $scope.numberNonexistentIncompleteCollections = (
+          responseData.number_of_nonexistent_activities.incomplete_collections
         );
-        $scope.numberDeletedCompletedExplorations = (
-          responseData.number_of_deleted_activities.completed_explorations
+        $scope.numberNonexistentCompletedExplorations = (
+          responseData.number_of_nonexistent_activities.completed_explorations
         );
-        $scope.numberDeletedCompletedCollections = (
-          responseData.number_of_deleted_activities.completed_collections
+        $scope.numberNonexistentCompletedCollections = (
+          responseData.number_of_nonexistent_activities.completed_collections
         );
-        $scope.numberDeletedExplorationPlaylist = (
-          responseData.number_of_deleted_activities.exploration_playlist
+        $scope.numberNonexistentExplorationPlaylist = (
+          responseData.number_of_nonexistent_activities.exploration_playlist
         );
-        $scope.numberDeletedCollectionPlaylist = (
-          responseData.number_of_deleted_activities.collection_playlist
+        $scope.numberNonexistentCollectionPlaylist = (
```

Comment:
Do you think we should change `numberNonexistentCollectionPlaylist` to `numberNonexistentCollectionsFromPlaylist` here and elsewhere so it reads better?

Reply:
Done!
should i change .
number_of_nonexistent_activities.exploration_playlist ==> number_of_nonexistent_activities.exploration_from_playlist ?